Add html and css design model based on bootstrap 4

areguera opened the pull request.

areguera committed:
Add html and css design model based on bootstrap 4

arrfab commented:
Looks good, but maybe we can (for now) still use the previous logo (instead of centos-logo-white.png) and also favicon.ico instead of favicon.png ?

areguera commented:
@arrfab centos-logos-white.png is the same image we use in wiki.centos.org, just with a different name to describe it is for use over dark backgrounds. Note that we have design differences between the logo in wiki.centos.org and the logo in headers. Which one should we use?

I am preparing a new commit to remove favicon.png an use favicon.ico instead. Plus the html corrections to make it visible.

areguera committed:
Use favicon.ico instead of favicon.png

arrfab commented on files/centos-design/header.html:
```diff
@@ -0,0 +1,86 @@
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <!-- CentOS customization -->
+    <link rel="stylesheet" href="css/centos.css">
```
shouldn't it be /centos-design/css/centos.css ? all files would land in that path on httpd server, so should be referenced with full path

areguera replied:
Yes. I am preparing the fix.

arrfab replied:
Thanks, looks good

areguera committed:
Update path to css and image files

- This commit also adds the "CentOS" word to CentOS logo image, as
  alternative text. In case images are not displayed (e.g., because the
  user disabled them in the web browser), the alternative text will.

arrfab commented:
Merging it in staging, but the footer.html will probably not be used, also with static caroussel with all sponsors (and it's always changing for centos.org itself, through different git repo) but at least that shows good ideas on what/how to use it to build a page with header + footer for dynamic content.
Thanks a lot !
